app/storage/encrypted_storage.py

```diff
@@ -5,80 +5,58 @@
 from app import settings
 from app.cryptography.jwe_decryption import JWEDirDecrypter
 from app.cryptography.jwe_encryption import JWEDirEncrypter
-from app.storage.abstract_server_storage import AbstractServerStorage
+from app.storage.database_storage import DatabaseStorage
 from app.utilities.strings import to_bytes
 from app.utilities.strings import to_str
 
 logger = logging.getLogger(__name__)
 
 
-class EncryptedServerStorageDecorator(AbstractServerStorage):
+def _safe_logging(msg, param):
```

Comment:
Shall we get rid of the logging if it's not used in production. It only adds risk to logging if misconfigured. In theory we can run without encryption locally